lib/chef/provider/aws_rds_instance.rb

```diff
@@ -36,14 +36,12 @@ def destroy_aws_object(instance)
     end
     # Wait up to 10 minutes for the db instance to shutdown
     converge_by "waited until RDS instance #{new_resource.name} was deleted" do
-      wait_for(
-        aws_object: instance,
-        query_method: :exists?,
-        expected_responses: [false],
-        acceptable_errors: [AWS::RDS::Errors::DBInstanceNotFound],
-        tries: 60,
-        sleep: 10
-      )
+      tries, sleep = 60, 10
```

Comment:
@josb https://github.com/nfedyashev/retryable#description

Nope - because we catch `acceptable_errors` and squelch them (don't reraise), the retryable block will exit.  Have you seen different behavior?  The tests for this were passing for me locally and not taking the full timeout (10 minutes).